### ISSUE TITLE
The free version cannot view images

### ISSUE THREAD

**Issue by freshman-x** [adrianboguszewski/debug-image-viewer#1]
The free version does not have the features in readme.

The free version of "View as Image" is not clickable.

**Comment by adrianboguszewski** [comment]
@freshman-x please let me know what image you're trying to show. Are we talking about Python? Is it a numpy array?

**Comment by freshman-x** [comment]
<img width="1104" alt="image" src="https://github.com/user-attachments/assets/5bf4b97e-3457-45b0-8e97-458dd611b4a1">
I want to display a tensor object in Python, but before I activated the free trial, the "View as Image" button was grayed out, and it showed a "premium" label.

(Just like in the image above, since I activated the free trial, I can't reproduce the issue right now. The image was Photoshopped to demonstrate the problem.)

**Comment by adrianboguszewski** [comment]
Thanks for the details. The plugin allows showing a numpy array (OpenCV image) for free. The support for Pillow images, TensorFlow and PyTorch tensors is available in the premium version as well as other features (https://plugins.jetbrains.com/plugin/14371-debug-image-viewer-former-opencv-image-viewer-/features). 

So please continue using your free trial and when it ends and you don't want to subscribe you can always create a watch, where you convert your tensor to a numpy array in the format of HxWxC, supported in the free version.

**Comment by freshman-x** [comment]
Thank you for your answer. nice work.
